debezium-core/src/main/java/io/debezium/relational/TableSchema.java

```diff
@@ -55,23 +56,33 @@
     /**
      * Create an instance with the specified {@link Schema}s for the keys and values, and the functions that generate the
      * key and value for a given row of data.
-     * 
+     *
+     * @param parentSchemaName the name of the schema; may be null
```

Comment:
Hmm... "parent" is still strange, especially because it maps to the envelope. Anyway, doesn't quite feel as clean.